### ISSUE TITLE
[Ruby] 2.3 de-indenting or "squiggly" heredoc syntax

### ISSUE THREAD

**Issue by grantovich** [sublimehq/Packages#189]
Ruby 2.3 introduces a de-indenting heredoc syntax:

```
doc_string = <<~INDENTED
  This is a multi-line string
    that will be automatically de-indented
      as per the indentation of the least-indented line
INDENTED
```

Sublime doesn't recognize `<<~` as the start of a string, causing the highlighting to go off the rails pretty quickly (in particular, if you have an odd number of single or double quotes within the heredoc, the entire remainder of the file will be highlighted as a string).

**Comment by wbond** [comment]
This should be resolved by 9c989b6c3d6553d535c2b5ed69d4f05f96534c16

**Comment by grantovich** [comment]
Awesome :ok_hand: Thanks @wbond!
